contrib/pkg/createcluster/create.go

```diff
@@ -105,15 +108,15 @@ type Options struct {
 	HiveImage          string
 	InstallerImage     string
 	ReleaseImage       string
+	ReleaseImageSource string
 	UseClusterImageSet bool
 	ManageDNS          bool
 	Output             string
 	IncludeSecrets     bool
 }
 
 const (
-	defaultHiveImage    = "registry.svc.ci.openshift.org/openshift/hive-v4.0:hive"
-	defaultReleaseImage = "registry.svc.ci.openshift.org/openshift/origin-release:v4.0"
+	defaultHiveImage = "registry.svc.ci.openshift.org/openshift/hive-v4.0:hive"
```

Comment:
Is our image still updated at this location, just the origin-release that isn't?

Reply:
That’s correct. It occurred to me this morning that there might be a 4.1 image stream now. Will check.